tutorial/20_recipes/005_user_defined_sampler.py

```diff
@@ -74,7 +74,7 @@ def sample_relative(self, study, trial, search_space):
         # the objective function passed to the study.
         params = {}
         for param_name, param_distribution in search_space.items():
-            if not isinstance(param_distribution, optuna.distributions.UniformDistribution):
+            if not isinstance(param_distribution, optuna.distributions.FloatDistribution):
```

Comment:
Should we add the same assertions as https://github.com/optuna/optuna-examples/pull/97/files to this tutorial?

Reply:
Right. Added in fbe336d.